configs/milvus.yaml

```diff
@@ -68,7 +68,9 @@ minio:
   port: 9000 # Port of MinIO/S3
   accessKeyID: minioadmin # accessKeyID of MinIO/S3
   secretAccessKey: minioadmin # MinIO/S3 encryption string
-  useSSL: false # Access to MinIO/S3 with SSL
```

Comment:
let's keep the old usessl config.
There is no need to offer cert file even if useSSL is enabled.
In order to maintain compatibility,
I would recommend to use  minio.useSSL + minio.ssl.cacert as configs